components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -11,12 +11,23 @@ import meh from "../images/feedback/meh-emoji.svg";
 import crying from "../images/feedback/crying-emoji.svg";
 import { trackEvent } from "../Analytics";
 
-function FeedbackComponent(props: { onClose: () => void; onSubmit: () => void; isModal: boolean }) {
+function FeedbackComponent(props: {
+    onClose?: () => void;
+    isModal: boolean;
+    isError: boolean;
+    message?: string;
+    initialSize?: number;
+}) {
     const [text, setText] = useState<string>("");
     const [selectedEmoji, setSelectedEmoji] = useState<number | undefined>();
+    const [isFeedbackSubmitted, setIsFeedbackSubmitted] = useState<boolean>(false);
 
-    const height = props.isModal ? "300px" : "";
-
+    const onClose = () => {
+        if (props.onClose) {
+            props.onClose();
+        }
```

Comment:
Can you use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls) here?

Reply:
Thanks for the review, @andrew-farries! I tried that but the linter still didn't like it 🙈